.github/workflows/generated-linux-aarch64-binary-manywheel-nightly.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       #       favor of GPU_ARCH_VERSION
       DESIRED_CUDA: cpu
       GPU_ARCH_TYPE: cpu-aarch64
-      DOCKER_IMAGE: pytorch/manylinux2_28_aarch64-builder:cpu-aarch64-main
+      DOCKER_IMAGE: pytorch/manylinux2_28_aarch64-builder-cpu-aarch64:f8555c14c97c7831a7f9e6eb8220b15ecbc8cb40
```

Comment:
Hi Catherine I believe it should be something like, include ecr url, its important:
```
308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/manylinux2_28-builder-cuda12.6:f8555c14c97c7831a7f9e6eb8220b15ecbc8cb40
```